hellodagger/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
nitpick: you could use [FragmentContainerView](https://developer.android.com/reference/androidx/fragment/app/FragmentContainerView)

Reply:
Okay, changed it to use FragmentContainerView.